### PR TITLE
es_generator clean up pass

### PR DESCRIPTION
**Pull request by muenstc**

### CODE REVIEW

File: source/MaterialXGenEssl/EsslShaderGenerator.cpp [hunk 1]
```diff
@@ -12,30 +12,21 @@ namespace MaterialX
...
-    // Temporary overwrites for THREE js
+    // Temporary overwrites for three.js
     _tokenSubstitutions[HW::T_IN_POSITION] = "position";
     _tokenSubstitutions[HW::T_IN_NORMAL] = "normal";
     _tokenSubstitutions[HW::T_IN_TEXCOORD] = "uv";
     _tokenSubstitutions[HW::T_IN_TANGENT] = "tangent";
-
-    // Register override implementations
-    // <!-- <image> -->
-    registerImplementation("IM_image_float_" + EsslShaderGenerator::TARGET, HwImageNode::create);
```
Comment (kohakukun) [inline source/MaterialXGenEssl/EsslShaderGenerator.cpp:33]: What was this doing?. Something with the texture images?

Comment (kohakukun) [reply source/MaterialXGenEssl/EsslShaderGenerator.cpp:33]: Looking further, I guess something that had to do with the shaders we are deleting? If so, what happened with the `IM_image_float_`. Just want to make sure there is no leftover somewhere

File: source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx [hunk 2]
```diff
@@ -1,29 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <materialx version="1.38">
-  <!-- Implementation overrides for ESSL -->
+  <!--
```
Comment (kohakukun) [inline source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx:3]: Is this file necessary at all? it is completely empty

File: source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx [hunk 3]
```diff
@@ -1,29 +1,9 @@
...
+    TM & (c) 2021 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
```
Comment (kohakukun) [inline source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx:4]: Not sure whether to put lucas films or Autodesk like in this file: resources/Materials/TestSuite/translation/standard_surface_to_UsdPreviewSurface.mtlx

File: source/MaterialXTest/CMakeLists.txt [hunk 4]
```diff
@@ -126,6 +126,11 @@ endif()
 add_custom_command(TARGET MaterialXTest POST_BUILD
 	COMMAND ${CMAKE_COMMAND} -E copy_directory
 	${CMAKE_CURRENT_SOURCE_DIR}/../../libraries ${CMAKE_BINARY_DIR}/bin/libraries)
+if(MATERIALX_BUILD_GEN_ESSL)
```
Comment (kohakukun) [inline source/MaterialXTest/CMakeLists.txt:129]: What difference does this change make?

Comment (muenstc) [reply source/MaterialXTest/CMakeLists.txt:129]: This will copy over the files (target definition, potential essl pbrlib/stdlib overrids) from the MaterialXGenEssl/libraries folder into the bin/libraries dir so that when you start MaterialXView it has access to those.

File: source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx (hunk 2 above)
Comment (muenstc) [reply source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx:3]: Just a stub, in case something gets added here later.

File: source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx (hunk 3 above)
Comment (muenstc) [reply source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx:4]: Good question, I copied it over from this file: libraries/stdlib/genglsl/stdlib_genglsl_impl.mtlx
@bernardkwok what should we do here?

File: source/MaterialXGenEssl/EsslShaderGenerator.cpp (hunk 1 above)
Comment (muenstc) [reply source/MaterialXGenEssl/EsslShaderGenerator.cpp:33]: I assume this was just to demonstrate how we could override specific nodes. As there are no changes to the base glsl implementation required to work for essl, I removed it again. @bernardkwok added those initially, was there another purpose to those?

File: source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx (hunk 2 above)
Comment (kohakukun) [reply source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx:3]: I would remove it. No future proofing . Then we don't need to answer the question of the copyright either 😬

File: source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx (hunk 3 above)
Comment (muenstc) [reply source/MaterialXGenEssl/libraries/stdlib/essl/stdlib_essl_impl.mtlx]: FYI: Removed the file.

**Review by kohakukun — APPROVED** [review]
